client/ratings_reviews/components/ReviewsList.jsx

```diff
@@ -47,6 +46,7 @@ class ReviewsList extends React.Component {
       });
     } else {
       this.setState({
+        allReviews: nextProps.reviews,
```

Comment:
Mostly out of curiosity, what is nextProps? Couldn't figure our where it's coming from!

Reply:
I'm sure there are more efficient ways to accomplish this, but apparently React doesn't always re-render the child even if the state of the parent has changed. This seems to be because React only re-renders the child if it detects the change in the child itself, not the parents, and sometimes React chooses to ignore subtle changes. "componentWillReceiveProps" seems to have fixed this issue for me, and "nextProps" simply refers to any subsequent props sending from the parent to the child after the initial "componentDidMount". I assume that it's similar to "componentDidUpdate" but only this one works for some reasons. I will do more investigation and get back to you! Also, my understanding is that React Hooks makes this process much easier? @jabrake